src/progressively.js

```diff
@@ -63,24 +73,56 @@
     } while (el !== document.body)
 
     return top <= document.documentElement.clientHeight
-  };
+  }
 
-  function loadImage (el) {
+/**
+ * Load image and add loaded-class. Loads the minified version, if small display
+ * @param  object DOMElement
+ * @param  object defaults
+ * @return boolean true, if fully loaded; false, if minified version was loaded
+ */
+  function loadImage (el, defaults) {
     setTimeout(function () {
       var img = new Image()
 
       img.onload = function () {
         el.classList.remove('progressive--not-loaded')
         el.classList.add('progressive--is-loaded')
-        el.src = this.src
+
+        if (el.classList.contains('progressive__bg')) {
+// Load image as css-background-image
+          el.style['background-image'] = 'url("' + this.src + '")'
+        } else {
+          el.src = this.src
+        }
 
         onLoad(el)
       }
 
+// Load minified version, if viewport-width is smaller than defaults.smBreakpoint:
+      if (getClientWidth() < defaults.smBreakpoint && el.getAttribute('data-progressive-sm')) {
+        el.classList.add('progressive--loaded-sm')
+        img.src = el.getAttribute('data-progressive-sm')
+        return false
```

Comment:
I'm not sure why you've used `true` and `false` as a value for checking whether the small is loaded or not. This makes it quite ambiguous about what the function intents to achieve.
But it will work for now maybe.